recipes/librealsense/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class LibRealSenseConan(ConanFile):
+    name = "librealsense"
+    description = "Cross-platform library for Intel RealSense depth cameras (D400 series and the SR300) " \
+                  "and the T265 tracking camera."
+    homepage = "https://github.com/IntelRealSense/librealsense"
+    topics = "conan", "intel", "3d", "computer vision", "robotics"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openmp": [True, False]
+    }
+    default_options = {"shared": False, "fPIC": True, "with_openmp": False}
+    generators = ["cmake", "cmake_find_package"]
+    requires = ["lz4/1.9.3", "boost/1.74.0"]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.options.with_openmp:
+            self.output.warn(
+                "Conan package for openmp is not available, this package will be used from system."
+            )
+
+    def requirements(self):
+        if self.settings.os != "Windows":
+            self.requires("libusb/1.0.23")
```

Comment:
Why is libusb not needed on Windows?

Btw, the libusb library on Linux is `libusb.a` (so linkable in cmake by adding `usb` to `target_link_libraries`).
On MSV it is called `libusb.lib`, so there it should be `target_link_libraries(... libusb)`.